osu.iOS/IOSMouseSettings.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Localisation;
+using osu.Game.Configuration;
+using osu.Game.Localisation;
+using osu.Game.Overlays.Settings;
+
+namespace osu.iOS
+{
+    public class IOSMouseSettings : SettingsSubsection
```

Comment:
not sure about this prefix. looks really wrong. can probably just be dropped?

Reply:
Renaming `AndroidMouseSettings` settings to `MouseSettings` means that `Game.Overlays.Settings.Sections.Input.MouseSettings.SensitivitySetting` has to be used to reference the sensitivity slider.

Unless you were saying this specifically about iOS, but the sensitivity slider will be used there in the future.

Or do you want it to be named `MouseSettings` until iOS specific settings are added?
Then we have two `MouseSettings`, one for iOS, one for desktop. That is more confusing IMO.

Reply:
I'd also be okay with `iOSMouseSettings` if we can make that work (also renaming `OsuGameIOS` to `OsuGameiOS`) but not sure if that is feasible.

Reply:
Rider by default doesn't like `iOSMouseSettings` (naming rule warning), and treats `OsuGameiOS` as a typo.

This config does make it work. Unfortunately, setting `iOS` as suffix won't fix the typo.
If this is a direction we want to take, we'd have to apply it framework side as well.

![image](https://user-images.githubusercontent.com/16479013/150008422-00cb4af0-08bb-4f5e-b226-bb6dfadc8f6c.png)

Reply:
I agree with `IOS` over `iOS`, and `MacOS` over `macOS`.

Reply:
I also agree with that as it reads better for class names and is also simpler for resharper to inspect (rather than `iOS` which requires explicit naming/typo rules).

Reply:
Okay I'll have to turn a blind eye